CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # New Relic Ruby Agent Release Notes
 
+## v9.0.0
+
+  Version 9.0.0 of the agent enables thread tracing.
+
+- **Enable Thread Instrumentation by default**
+
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This will allow the agent to properly monitor code that occurs inside of threads. If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists.
```

Comment:
Should we specify "Ruby Threads"? And maybe note that we don't have instrumentation for Java threads for users of JRuby, etc.?

Reply:
The thread class instrumentation works in jruby as well though. If you are using jruby and create a thread using Thread.new, that will be instrumented just like cruby is. 
I think what you're referring to is concurrent ruby because concurrent ruby _doesn't_ use the Thread class when it's running jruby, it does something different related to java specific stuff. Similarly in cruby though, if someone creates threads using c instead of the ruby thread class, we wouldn't have instrumentation for that either.

Reply:
Thanks for setting me straight! That makes much more sense!